packages/flutter/test/material/text_field_test.dart

```diff
@@ -41,12 +40,12 @@ void main() {
   SystemChannels.platform.setMockMethodCallHandler(mockClipboard.handleMethodCall);
 
   const String kThreeLines =
-    'First line of text is here abcdef ghijkl mnopqrst. ' +
-    'Second line of text goes until abcdef ghijkl mnopq. ' +
-    'Third line of stuff keeps going until abcdef ghijk. ';
+    'First line of text is ' +
```

Comment:
I assume that all of the abcd... stuff was to force the text field to wrap. Is that not needed anymore?